server/planarserver.py

```diff
@@ -3,6 +3,14 @@
 This is the code responsible for starting the backend and reacting to socket IO events.
 """
 
+# Check for existence of './templates/' as it is not present if client was not built before
+import sys
+from utils import FILE_DIR
+
+if( (FILE_DIR / "templates").exists() is False):
```

Comment:
```suggestion
if not (FILE_DIR / "templates").exists():
```
Another small nitpick, it's more pythonic to use not here instead of an is False.  also there is no need to add surrounding parentheses in python

Reply:
Thanks! :D